build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -3,8 +3,8 @@
 
   <PropertyGroup>
     <StartAction Condition="'$(StartActions)' == ''">Program</StartAction>
-    <StartProgram Condition="'$(StartProgram)' == ''">$(MSBuildThisFileDirectory)\..\..\..\Closed\Tools\xUnit\xunit.gui.custom.exe</StartProgram>
-    <StartArguments Condition="'$(StartArguments)' == ''">$(AssemblyName).dll</StartArguments>
+    <StartProgram Condition="'$(StartProgram)' == ''">$(MSBuildThisFileDirectory)\..\..\packages\xunit.runner.console.$(xUnitVersion)\tools\xunit.console.x86.exe</StartProgram>
```

Comment:
It should not block merging this PR, but we probably need a better story for this for folks who use xunit.gui.custom today (like maybe check to see if it exists in the OutputDir or somewhere that people can copy it if they want it).